modules/workload-identity/output.tf

```diff
@@ -40,6 +40,6 @@ output "gcp_service_account_name" {
 }
 
 output "gcp_service_account" {
-  description = "GCP service account."
+  description = "GCP service account, if created; null if existing account was used."
```

Comment:
We should always pass this output back. If an existing SA is used, just reflect that value back.

Reply:
Is there a way to lookup an existing GCP service account as a Terraform object in order to return it? I return all the other (string) output fields, but this one is the entirely (newly created) object...

Reply:
Yes, you can use the data source: https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/service_account

Reply:
Ah, nice! Something like the most recently pushed change, then?

Reply:
Yep, you'll still need to update the output to point to the data source.

Reply:
I think I've done that... because I've used `main` as the name of the resource, to be consistent with the KSA, that's left me with `cluster_service_account` as a name for the data source. Given that, is the output just the same, i.e. `google_service_account.cluster_service_account`?

Reply:
No. When the data source is used, you need to refer to it as `data.google_service_account.cluster_service_account`.